solutions/cses-1091.mdx

```diff
@@ -51,3 +54,124 @@ int main(){
 	return 0;
 }
 ```
+
+</CPPSection>
+
+<JavaSection>
+```java
+import java.io.*;
+import java.util.*;
+
+public class ConcertTickets {
+	public static void main(String[] args) throws IOException {
+		Reader io = new Reader();
+		PrintWriter pw = new PrintWriter(System.out);
+
+		int ticketNum = io.nextInt();
+		int peopleNum = io.nextInt();
+		// No multiset in java so we'll have to use a TreeMap
+		NavigableMap<Integer, Integer> ticketMultiset = new TreeMap<>();
+		Map.Entry<Integer, Integer> val;
+
+		for (int i = 0; i < ticketNum; i++) {
+			int priceOfTicket = io.nextInt();
+			/*
+			 * If the set already contains the same key value, 
+			 * create another pair next to it
+			 */
+			if (ticketMultiset.containsKey(priceOfTicket)) {
+				ticketMultiset.put(priceOfTicket, 
+					      ticketMultiset.get(priceOfTicket) + 1);
```

Comment:
```suggestion
				ticketMultiset.put(priceOfTicket, 
									ticketMultiset.get(priceOfTicket) + 1);
```
does this look better?
![image](https://user-images.githubusercontent.com/63619830/129510461-f8d9fe61-cd43-420d-9014-25caf276d4b3.png)